### PR TITLE
Add rate limiter dependency into Tmail

### PR DESCRIPTION
**Pull request by Arsnael**

### CODE REVIEW

**Comment by Arsnael** [comment]
It works, as I could check locally. However, I was not sure if it was the best way or if we can do something smarter with maven here..?

File: tmail-backend/apps/memory/pom.xml [hunk 1]
```diff
@@ -187,6 +187,39 @@
...
+                        <id>copy-rate-limiting-jar</id>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>copy-resources</goal>
+                        </goals>
+                        <configuration>
+                            <outputDirectory>${basedir}/target/extensions-jars</outputDirectory>
+                            <resources>
+                                <resource>
+                                    <directory>../../../james-project/server/mailet/rate-limiter-redis/target</directory>
```
Comment (chibenwa) [inline tmail-backend/apps/memory/pom.xml:200]: Let's use the in memory one only, no?

**Review by chibenwa — APPROVED** [review]
Ok if it works though adding a regular maven depency would have had worked too.

We could also register the rate limiters guice mudule directly in the guice bindings of the APP, so configuration is eased.

**Comment by Arsnael** [comment]
> Ok if it works though adding a regular maven depency would have had worked too.
> 
> We could also register the rate limiters guice mudule directly in the guice bindings of the APP, so configuration is eased.

Hmmm I think I misunderstood. It might be easier and makes more sense indeed. Clearly looked for complicated for nothing here

**Comment by Arsnael** [comment]
Needs https://github.com/apache/james-project/pull/883

**Review by quantranhong1999 — APPROVED** [review]

**Review by chibenwa — APPROVED** [review]

**Review by vttranlina — APPROVED** [review]

**Pull request merged**